modules/Bio/EnsEMBL/Production/Pipeline/TSV/DumpFileXref.pm

```diff
@@ -130,7 +130,7 @@ sub _write_tsv {
     my $unzip_out_file = $out_file;
     `gzip -n $unzip_out_file`;
 
-    if (-e $unzip_out_file) { `rm $unzip_out_file`; }
+    if (-e $unzip_out_file) { die "gzip failed"; }
```

Comment:
May be, we should still delete the file? then `die` ?

Reply:
Just checking, is the `-e` not testing for the file to exists, then it will die if successful?

Reply:
If the file exists, then gzip failed.
Gzips standard usage
$gzip file.txt
$ls
file.txt.gz

Reply:
Ah ok, my mistake! I misread: you were testing the actual zip file presence. All good then,